mongodbatlas/data_source_mongodbatlas_advanced_clusters.go

```diff
@@ -221,7 +223,11 @@ func flattenAdvancedClusters(ctx context.Context, d *schema.ResourceData, conn *
 	results := make([]map[string]interface{}, 0)
 
```

Comment:
[q] why `d *schema.ResourceData` is an argument of this function if it is not used?

Reply:
Good eye, will remove the argument since is not necessary this time